internal/ingress/controller/template/configmap_test.go

```diff
@@ -31,6 +31,22 @@ func TestFilterErrors(t *testing.T) {
 	}
 }
 
+func TestProxytTimeoutParsing(t *testing.T) {
+	testCases := map[string]struct {
+		input  string
+		expect float64 // duration in seconds
```

Comment:
Same suggestion, use `time.Duration` instead of assuming we use seconds and set the `expect` field to something like `time.Duration(5) * time.Second` in your test cases. This way a test case like `{ "100ms",  time.Duration(100) * time.Millisecond }` is both valid and readable.